packages/core-pricing/plugins/product-price-coinbase-exchange.js

```diff
@@ -0,0 +1,71 @@
+import fetch from 'isomorphic-unfetch';
+import {
+  ProductPricingDirector,
+  ProductPricingAdapter,
+} from 'meteor/unchained:core-pricing';
+import { Countries } from 'meteor/unchained:core-countries';
+import Cache from '../utils/cache';
+
+const CACHE_PERIOD = 60 * 60 * 0.1; // 10 minutes
+const SUPPORTED_CURRENCIES = [
+  'BTC',
+  'ETH',
+  'XRP',
+  'USDT',
+  'BCH',
+  'BSV',
+  'LTC',
+  'EOS',
+  'BNB',
+  'XTZ',
+];
+const cache = new Cache(CACHE_PERIOD);
+
+const getFiatexchangeRateForCrypto = async (base, target) => {
+  const { data } = await cache.get(`${base}-${target}`, () =>
```

Comment:
ECB rates can be cached for 24h I thought, maybe define here specifically rather than the default 60s